target/linux/ramips/image/mt7621.mk

```diff
@@ -171,6 +171,7 @@ endef
 define Build/zytrx-header
 	$(eval board=$(word 1,$(1)))
 	$(eval version=$(word 2,$(1)))
+	$(eval version=$(shell echo $(version) | cut -c 1-31))
```

Comment:
> zytrx accepts version strings up to 32 characters.

So why only cut 31 characters?

Reply:
Honestly no idea, this PR is one year old. But  I guess its a off-by-one error on my part.